packages/react-native-platform-override/src/Cli.ts

```diff
@@ -382,36 +397,46 @@ async function checkFileExists(friendlyName: string, filePath: string) {
 /**
  * Read a manifest and print a pretty error if we can't
  */
-async function readManifest(file: string, version?: string): Promise<Manifest> {
-  const gitReactRepo = await GitReactFileRepository.createAndInit();
-  const ovrRepo = new OverrideFileRepositoryImpl(path.dirname(file));
+async function readManifest(file: string): Promise<Manifest> {
+  try {
+    const data = await Serialized.readManifestFromFile(file);
```

Comment:
Having these scoped to a `Serialized` module is a bit awkward. Consider changing serialized stuff to ES6 exports and add naming a different way.

Reply:
Going to leave this as is